### PR TITLE
docs: add INSTALL.md

### LINKED ISSUE #24: Add INSTALL.md to project
According to TRG 1.02 a repository should have an INSTALL.md file and TRG 2.03 lists it as well.
https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-2
https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3

### PR DESCRIPTION
**Pull request by borisrizov-zf**
## Description

Although TRG 1.02 doesn't require the INSTALL.md unless the project has a lengthy installation, TRG 2.03 suggests that the file is present.
Closes #24 

**NB** I used the opportunity to format the README.md using the newly setup rules.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ x ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ x ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by florianrusch-zf** [comment]
:D Should we really add an empty file? Maybe we can add at least one sentence, or some entrypoint links (e.g. to the helm chart)

**Comment by borisrizov-zf** [comment]
> :D Should we really add an empty file? Maybe we can add at least one sentence, or some entrypoint links (e.g. to the helm chart)

I'm torn myself actually... 

No Helm yet, as this is not meant to be deployed anywhere, it's just a lib.

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -124,63 +147,69 @@ To build the project from source, follow these steps:
 
 4. After a successful build, you can include the generated JAR file in your project's dependencies.
```
[the remark this answers is not in the document]
Comment (borisrizov-zf) [reply README.md:148]: It's included a few lines up, right before the `Build from Source` instruction.

**Comment by borisrizov-zf** [comment]
@florianrusch-zf ping

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  <artifactId>cx-ssi-lib</artifactId>
+  <version>0.0.17</version>
+  <scope>compile</scope>
+</dependency>
+```
+
+# Install for Gradle project
+
+Add a custom maven repository to your `build.gradle`
+
```
[the remark this answers is not in the document]
Comment (borisrizov-zf) [reply INSTALL.md:65]: done

**Review by florianrusch-zf — APPROVED** [review]

**Pull request merged**